### DISCUSSION TITLE
dotnet/client.py comment that EncryptionAndAuthentication won't work in Bleak

### DISCUSSION THREAD

**Discussion by sigsbeeus** [hbldh/bleak#636] [Q&A]
In the comments for the pair() function in the dotnet backend, client.py, there's an assertion that the EncryptionAndAuthentication protection_level won't work in Bleak.   We have a peripheral device we're developing and would like to use Bleak on Windows 10 for testing.   We aren't getting anywhere trying to modify the source to support EncryptionAndAuthentication.    Can anyone tell me the reason the comment is there?  Is there something about Windows that prevents python from implementing that protection_level?  Thank you.

**Comment by hbldh** [comment]
Sorry for late answer, but I cannot remember why right now. It is more probable that I had no device to test this on, so I wrote something to just communicate that this setting is not supported by Bleak right now. You are more than welcome to try to implement it if you have a device that supports such pairings!
